.gitlab-ci.yml

```diff
@@ -1,70 +1,16 @@
 
 stages:
-  - build
-  - publish
   - deploy
-  
+  - deploy_aas
+
 variables:
   DATADOG_AGENT_WINBUILDIMAGES: v5996510-a9e6a7d
   GIT_PROFILER_REF: master
   DEPLOY_TO_REL_ENV:
     value: "false"
     description: "Set to true to override rules in the reliability-env pipeline (e.g. override 'only deploy master')"
-    
-build:
-  only:
-    - master
```

Comment:
Won't this remove our gitlab builds entirely? We need those to sign the .NET library releases.

Reply:
Oh, looking at the individual commits, it looks like @pierotibou removed this temporarily while testing. I'm gonna leave the "request changes" to avoid merging by mistake :)